src/problem-analysis/state-of-the-art/lichess-org.tex

```diff
@@ -1,4 +1,38 @@
 % textidote: ignore begin
 \subsection{Lichess.org}\label{subsec:lichess-org}
+
+\subsubsection{What is Lichess.org?}\label{subsubsec:what-is-lichess-org}
```

Comment:
Maybe delete this line and just make the "Lichess.com" the subsection title. I dont know though, just a thought